client/src/components/Review.jsx

```diff
@@ -68,7 +68,7 @@ export function Review(props) {
                   Posted by: {props.username}
                 </Typography>
               </div>
-              {user.username === props.username && (
+              {user?.username === props.username && (
```

Comment:
@nidhikay added this. Was getting an error when viewing this information while not logged in.